engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/details/repository-details-info.controller.js

```diff
@@ -110,7 +110,22 @@ angular.module('Bastion.repositories').controller('RepositoryDetailsInfoControll
                 } else {
                     repository["docker_tags_whitelist"] = [];
                 }
-                /* eslint-disable camelcase */
+              /* eslint-disable camelcase */
+                if (repository.upstream_password === '') {
```

Comment:
Maybe an unnecessary optimization but it might be good to maintain a list of these that we want to check in this same manner...

```
var fields = ['upstream_password', ...]

angular.forEach(fields, function(field) {
   if (repository[field] === '') {
     repository[field] = null;
   }
});
```

Reply:
Also gets rid of all if blocks around this and makes change cleaner..Updated..